e2e/cypress/integration/messaging/collapse_link_spec.js

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Link preview - Removing it from my view removes it from other user\'s view', () => {
+    before(() => {
+        // # Set the configuration on Link Previews
+        setSystemConsoleLinkPreview();
```

Comment:
Please have it via API, like:
```javascript
cy.apiUpdateConfig({
    ServiceSettings: {
        EnableLinkPreviews: true,
    },
});
```

Such API is based from https://api.mattermost.com/#tag/system%2Fpaths%2F~1config%2Fput